SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -206,9 +206,25 @@ bool FLocalDeploymentManager::LocalDeploymentPreRunChecks()
 	return bSuccess;
 }
 
-void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FString RuntimeVersion, FString LaunchArgs,
-													  FString SnapshotName, FString RuntimeIPToExpose,
+void FLocalDeploymentManager::TryStartLocalDeployment(const FString& LaunchConfig, const FString& RuntimeVersion, const FString& LaunchArgs,
+													  const FString& SnapshotName, const FString& RuntimeIPToExpose,
 													  const LocalDeploymentCallback& CallBack)
+{
+	int NumRetries = 3;
+	while (NumRetries > 0)
+	{
+		NumRetries--;
+		ERuntimeStartResponse Response = StartLocalDeployment(LaunchConfig, RuntimeVersion, LaunchArgs, SnapshotName, RuntimeIPToExpose, CallBack);
```

Comment:
I think we want some exponential backoff here, otherwise we might retry too quickly.

Reply:
You mean a sleep after the thing times out?

Reply:
Yeah, with the length of the sleep doubling each time it gets hit

Reply:
Hmm I am not sure what length of time you are imagining for the sleep but it doesn't seem super necessary to me.
We only retry when the runtime start up times out. This already a timeout of 10 seconds. You think we will gain something waiting extra time?

Reply:
I agree with Matt, doesn't seem like extra waiting time is necessary.